### PR TITLE
Ma issue fixes ("Why Sonr" and "App structure")

### PR DESCRIPTION
**Pull request by ma-sonr**
## Changes

- [Application vs. Blockchain layer image](https://github.com/sonr-io/docs/issues/49)
- [Verifiable Identities in Why Sonr](https://github.com/sonr-io/docs/issues/48)

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Comment by ntindle** [comment]
Fixes #48

**Comment by ma-sonr** [comment]
Fixes #49

File: blog/2022-09-06-welcome/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 slug: welcome
 title: Welcome
-authors: [prad.snr/, michael.snr/]
+authors: [prad.snr/]
```
Comment (ntindle) [inline blog/2022-09-06-welcome/index.md:4]: Lol why tho

Comment (ma-sonr) [reply blog/2022-09-06-welcome/index.md:4]: Fixed this in ma-blog - I'll write Welcome article and he'll write Flutter

**Pull request closed without merging**